packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -344,6 +361,18 @@ const generateHelloWorld = ({projectId}, {outputDir}) => {
     })
 }
 
+const presetPrompt = () => {
+    const questions = [
+        {
+            name: 'preset',
+            message: 'Choose a project preset to get started:',
+            type: 'list',
+            choices: PUBLIC_PRESETS
+        }
+    ]
+    return inquirer.prompt(questions).then((answers) => answers['preset'])
```

Comment:
@vcua-mobify I was trying the new release and saw that we are displaying the preset ids, instead of the detailed textual options. After seeing this line, I believe this is so that we can automatically get all the available public presets, is that so?

Reply:
The prompt is using the same array of presets used elsewhere in the code. It is implemented in this way to keep the code straightforward and since the choices using IDs are easy to identify for a user who is reading.

We could add longer descriptions but they would need to be maintained in addition to the ids and the code will need a way to go from the longer description to the id. I am also not clear on what value the longer descriptions will add that the ids are not (in Slack, I recall there was a thread saying we don't want to explicitly mention backends in the choices).